README.md

```diff
@@ -100,6 +101,7 @@ track_written: false                     # Optional: track written stripes
 write_through: false                     # Optional: enable write-through mode
 autofetch: false                         # Optional: fetch stripes automatically
 device_id: "ubiblk"                      # Optional: device identifier
+io_engine: "io-uring"                    # Optional: io-uring (default) or sync
```

Comment:
`fio` uses the string `io_uring` (with underscore), so I have a slight preference to use it instead of `io-uring`.